.github/workflows/test.yml

```
@@ -0,0 +1,34 @@
+name: Test
+on: [push, pull_request]
```

```suggestion
on: push
```

Because secrets are not available to pull requests from a fork.